packages/react/src/Delay.spec.tsx

```diff
@@ -16,9 +24,25 @@ describe('<Delay/>', () => {
     render(<Delay>{TEXT}</Delay>)
     expect(screen.queryByText(TEXT)).toBeInTheDocument()
   })
-  it('should accept 0 for ms prop', async () => {
+  it('should accept 0 for ms prop', () => {
     render(<Delay ms={0}>{TEXT}</Delay>)
-    await waitFor(() => expect(screen.queryByText(TEXT)).toBeInTheDocument(), { timeout: 1000 })
+
+    expect(screen.queryByText(TEXT)).toBeInTheDocument()
+  })
+  it('should render fallback content initially and then the actual text after the delay', async () => {
+    render(
+      <Delay ms={ms('1s')} fallback={<p role="paragraph">fallback</p>}>
+        {TEXT}
+      </Delay>
+    )
+
+    vi.advanceTimersByTime(ms('0.5s'))
+
+    expect(screen.queryByRole('paragraph')).toBeInTheDocument()
+
+    vi.advanceTimersByTime(ms('0.5s'))
```

Comment:
```suggestion
    expect(screen.queryByRole('paragraph')).toBeInTheDocument()
    vi.advanceTimersByTime(ms('1s'))
```

Reply:
Fix done! 

Additionally, I followed the guidelines in `testing-library`. The changes are as follows

```ts
beforeEach(() => {
  vi.useFakeTimers({ shouldAdvanceTime: true })
})

afterEach(() => {
  vi.runOnlyPendingTimers()
  vi.useRealTimers()
})
```

### Related
https://testing-library.com/docs/using-fake-timers/